guides/theming-your-components.md

```diff
@@ -11,46 +11,152 @@ customize components. Angular Material provides APIs for reading values from thi
+| card                     | The background color of a card                     |
+| dialog                   | The background color of a dialog                   |
+| raised-button            | The background color of a raised button            |
+| selected-button          | The background color of a selected button          |
+| selected-disabled-button | The background color of a selected disabled button |
+| disabled-button          | The background color of a disabled button          |
+| focused-button           | The background color of a focused button           |
+| disabled-button-toggle   | The background color of a disabled button toggle   |
+| unselected-chip          | The background color of an unselected chip         |
+| disabled-list-option     | The background color of a disabled list option     |
+| tooltip                  | The background color of a tooltip                  |
+
+The foreground palette supports reading the following named colors:
+
+| Color name        | Description                                                                                              |
+|-------------------|----------------------------------------------------------------------------------------------------------|
+| base              | The base foreground color, can be used to for color mixing or creating a custom opacity foreground color |
+| divider           | The color of a divider                                                                                   |
+| dividers          | (Alternate name for divider)                                                                             |
+| disabled-text     | The color for disabled text                                                                              |
+| disabled          | (Alternate name for disabled-text)                                                                       |
+| disabled-button   | The color for disabled button text                                                                       |
+| elevation         | The color elevation shadows                                                                              |
+| hint-text         | The color for hint text                                                                                  |
+| secondary-text    | The color for secondary text                                                                             |
+| icon              | The color for icons                                                                                      |
+| icons             | (Alternate name for icon)                                                                                |
+| text              | The color for text                                                                                       |
+| slider-min        | TODO(mmalerba): what is this?                                                                            |
```

Comment:
Same for foreground, and for these slider ones in particular, I'm not sure what they were intended for and they seem to be used precisely nowhere in our code.